src/pci_slot.c

```diff
@@ -66,3 +62,98 @@ void pci_slot_fini(struct pci_slot *slot)
 		free(slot);
 	}
 }
+
+/**
+ * @brief Finds PCI slot by number of the slot.
+ *
+ * @param[in]       slot_number       Number of the slot
+ *
+ * @return Struct with pci slot if successful, otherwise the function returns NULL pointer.
+ */
+static struct pci_slot *find_pci_slot_by_number(char *slot_number)
```

Comment:
const char* slot?

Reply:
missing slot_numer will back to us on each static code analisis, so maybe assert(slot_number)?

Reply:
Or maybe let's just add at the beginning
```c
if (!slot_number)
   return NULL;
```